From 8ed3ecddf00e5734182dd5eb7af869e99f005876 Mon Sep 17 00:00:00 2001
From: TheKit <nekit1000@gmail.com>
Date: Sun, 21 Feb 2021 17:32:28 +0200
Subject: [PATCH 5/8] [ofono] mbpi: default to IPv4 during context provisioning

Using dual IPv4/IPv6 for internet contexts causes with some
cellular providers on Ubuntu Touch.

Related to https://github.com/ubports/ubuntu-touch/issues/1657
---
 ofono/plugins/mbpi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ofono-sailfish/ofono/plugins/mbpi.c
===================================================================
--- ofono-sailfish.orig/ofono/plugins/mbpi.c
+++ ofono-sailfish/ofono/plugins/mbpi.c
@@ -58,7 +58,7 @@ const char *mbpi_database = MBPI_DATABAS
  * Use IPv4 for MMS contexts because gprs.c assumes that MMS proxy
  * address is IPv4.
  */
-enum ofono_gprs_proto mbpi_default_internet_proto = OFONO_GPRS_PROTO_IPV4V6;
+enum ofono_gprs_proto mbpi_default_internet_proto = OFONO_GPRS_PROTO_IP;
 enum ofono_gprs_proto mbpi_default_mms_proto = OFONO_GPRS_PROTO_IP;
 enum ofono_gprs_proto mbpi_default_ims_proto = OFONO_GPRS_PROTO_IPV4V6;
 enum ofono_gprs_proto mbpi_default_proto = OFONO_GPRS_PROTO_IP;
